project_risk/views/project_risk_view.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="project_risk_form_view" model="ir.ui.view">
+        <field name="model">project.risk</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                    <field name="state" widget="statusbar" statusbar_visible="draft,active,closed" clickable="True"/>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only"/>
+                        <h1><field name="name"/></h1>
+                    </div>
+                    <group>
+                        <group>
+                            <field name="project_id"/>
+                            <field name="project_risk_category_id" widget="selection"/>
+                            <field name="owner_id"/>
+                            <field name="actionee_id"/>
+                        </group>
+                        <group>
+                            <field name="probability"/>
+                            <field name="impact"/>
+                            <field name="rating"/>
+                            <field name="proximity"/>
+                        </group>
+                    </group>
+                    <notebook>
+                        <page string="Description">
+                            <field name="description"/>
+                        </page>
+
+                        <page string="Response">
+                            <group>
+                                <field name="project_risk_response_category_id" widget="selection"/>
+                            </group>
+                            <field name="project_risk_response_ids">
```

Comment:
A `<form>` view is missing. When a user has no management permissions the default weird form is open when clicking the file record